incubator/hnc/docs/user-guide/quickstart.md

```diff
@@ -0,0 +1,651 @@
+```
+
+If you check existing secrets in `service-1` using the command below, you will
+find that the secret does not show up in `service-1` because we haven't
+configured HNC to propagate secrets in HNCConfiguration.
+
+```bash
+kubectl -n service-1 get secrets
+```
+
+In order to get this to work, you need to update the HNCConfiguration object,
+which is a single cluster-wide configuration for HNC as a whole. To do this,
+simply use the config subcommand:
+
+```bash
+# In HNC v0.6:
+kubectl hns config set-resource secrets --mode Propagate
+
+# In HNC v0.5:
+kubectl hns config set-type --apiVersion v1 --kind Secret propagate
+```
+
+  Note: In HNC v0.5, the apiVersion of _v1_ here applies to the Secret, and the word
+  _propagate_ refers to how we want HNC to treat this kind of object. In HNC
+  v0.6, the version is not required, but an optional `--group` parameter can
+  be used for resources that are not part of the core Kubernetes API.
+
+  Note: As of HNC v0.6, the supported modes are `Propagate`, `Remove` and
+  `Ignore`. More may be added in the future; see `kubectl hns config
+  set-resource` for the latest documentation.
+
+Now, we should be able to verify that `my-creds` was propagated to `service-1`:
+
+```bash
+kubectl -n service-1 get secrets
+```
+
+And finally, note that if we move the service back to team-a, the secret disappears because we haven't created it there:
+
+```bash
+kubectl hns set service-1 --parent team-a
+kubectl hns tree acme-org
+kubectl -n service-1 get secrets
+```
+
+If you like, you can also view the entire cluster-wide configuration for HNC by
+saying `kubectl hns config describe`. You can also look directly at the
+underlying cluster-wide configuration object:
+
+```bash
+kubectl get hncconfiguration config -o yaml
+```
+
+Which should show something like the following in HNC v0.6:
+
+```yaml
+apiVersion: hnc.x-k8s.io/v1alpha2
+kind: HNCConfiguration
+metadata:
+  name: config
+spec:
+  resources:
+  - resource: secrets
+    mode: Propagate
+status:
+  resources:
+  - group: rbac.authorization.k8s.io/v1
+    mode: Propagate
+    numPropagatedObjects: 4
+    numSourceObjects: 35
+    resource: role
+    version: v1
+  - group: rbac.authorization.k8s.io/v1
+    mode: Propagate
+    numPropagatedObjects: 4
+    numSourceObjects: 77
+    resource: rolebinding
+    version: v1
+  - mode: Propagate
+    resource: secrets
+    numPropagatedObjects: 2
+    numSourceObjects: 1
+    version: v1
+```
+
+  Note: HNC v0.5 uses the `v1alpha1` API, which enforces that the RBAC objects
+  are also shown in the spec. In `v1alpha2`, RBAC objects are only shown in the
+  status.
+
+You can also edit this object directly if you prefer, as an alternative to
+using the kubectl plugin - the object is created automatically when HNC is
+installed on the cluster.
+
+
+<a name="netpol"/>
+
+## Hierarchical network policy
+
+_Demonstrates: Network Policy integration via the 'tree' label._
```

Comment:
What does "integration via the 'tree' label." mean?

Reply:
Changed to "Making Network Policies hierarchy-aware using the 'tree' label"